extension/chrome/settings/modules/my_key_update.ts

```diff
@@ -54,6 +54,7 @@ View.run(class MyKeyUpdateView extends View {
   private storeUpdatedKeyAndPassphrase = async (updatedPrv: Key, updatedPrvPassphrase: string) => {
     const storedPassphrase = await PassphraseStore.get(this.acctEmail, this.primaryKi!.fingerprints[0], true);
     await KeyStore.add(this.acctEmail, updatedPrv);
+    // todo:
```

Comment:
should we leave this copying as it is or rely on OrgRules?
Is there a reason to save `undefined` to local storage (line 58) in case it was read as `undefined`  on line 55?

Reply:
I think this is also a good place to add a restriction based on OrgRules. Usually, if at this point "the bad thing will never happen", then you can make assertions by throwing an `Error` if whatever assumptions we have are false (which will cause these errors to be reported to us, so we can fix it).

If it was read as undefined from line 55, then there is no particular reason to store undefined back to storage (undefined is supposed to remove it from storage, but based on line 55, it's not there anyway).